src/dtos/duct-waf-base.dto.ts

```diff
@@ -52,13 +104,25 @@ export class DuctWafBaseDTO {
     name:
       propertyMetadata.ductWafDTONumberOfTraversePointsWaf.fieldLabels.value,
   })
+  @IsInt()
+  @IsAtMostDigits(2, {
+    message: (args: ValidationArguments) => {
+      return `${args.property} [RECTDUCTWAF-FATAL-A] The value for ${args.value} in the Rectangular Duct Waf record ${args.property} must be 5 digits or less`;
```

Comment:
change error message to 2 digits or less

Reply:
✔